packages/teams-js/src/private/files.ts

```diff
@@ -799,7 +799,10 @@ export namespace files {
    * @internal
    * Limited to Microsoft-internal use
    */
-  export function openDownloadFolder(fileObjectId: string = undefined, callback: (error?: SdkError) => void): void {
+  export function openDownloadFolder(
+    fileObjectId: string | undefined = undefined,
```

Comment:
> | undefined [](http://example.com/codeflow?start=24&length=12)

Obviously the type on this parameter must include `undefined` since it's literally being set to `undefined` as the default value here.